docs/starters.yml

```
@@ -1829,3 +1829,21 @@
     - Various available icon sets
     - RSS Feed
     - Web app manifest
+- url: https://gatsby.magicsoup.io/
+  repo: https://github.com/magicsoup-io/gatsby-starter-magicsoup
+  description: A production ready gatsby starter using magicsoup.io
+  tags:
+    - SEO
+    - Markdown
+    - Testing
+    - Styling:Styled-Components
+  features:
+    - Optimized images! No more large, unoptimized images which dramatically slow down your site. The starter uses gatsby-image for optimized image loading.
```

Can we drastically simplify these features? If you compare to some other starters--they are generally just the first part of these features, e.g.

```suggestion
    - Optimized images
```

Not as fun, I know--but generally more consistent!

---

Yes for sure. I will mention which starter or how tos I have implemented here and cut it down.

---

I removed the information about the images, because it's gatsby standard I would think.

---

The info about SSR is obsolet as well I think